src/account-info/ProfileScreen.js

```diff
@@ -45,6 +46,19 @@ function SetStatusButton(props: {||}) {
   );
 }
 
+function GoToSettingsButton(props: {||}) {
+  return (
+    <ZulipButton
+      style={styles.button}
+      secondary
+      text="Go to Settings"
```

Comment:
Let's just call it "Settings". It's shorter and gets the point across. 🙂 (Here and in the component's name; there may be other places.)

Reply:
Okay, will be changing to "Settings". Yeah, there are some places 1 or 2, where it need to be changed. I will do it 🙂.